src/coreclr/jit/emit.cpp

```diff
@@ -3782,6 +3782,20 @@ const IS_INFO emitter::emitGetSchedInfo(insFormat insFmt)
     assert(!"Unsupported insFmt");
     return IS_NONE;
 }
+
+bool emitter::HasApxPpx(instruction ins)
```

Comment:
can you add summary docs explaining what is the scenario in which this method is used and what inference we can make from it?

Reply:
Done